src/Tribe/Aggregator/Admin_Bar.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Class to initialize and add the required items in
+ *
+ * @since  TBD
+ */
+class Tribe__Events__Aggregator__Admin_Bar {
+
+	/**
+	 * @since  TBD
+	 *
+	 * @var Tribe__Events__Admin__Bar__Admin_Bar
+	 */
+	protected static $instance;
```

Comment:
Add a line break between property declarations and docblock comments after them, pretty please!

Reply:
@bordoni this is the last thing you need to fix before this can merge — looking good otherwise!

Reply:
This was fixed.